tests/integration/init/schemas/test_init_with_schemas_command.py

```diff
@@ -317,10 +332,3 @@ def test_init_interactive_with_event_bridge_app_non_supported_schemas_region(sel
             result = runner.invoke(init_cmd, ["--output-dir", temp], input=user_input)
             self.assertTrue(result.exception)
             self._tear_down_custom_config()
-
-
-def _get_command():
```

Comment:
Why remove this?

Reply:
That is not used, copy/paste from other test files I believe. I also changed this in the other file to use a centralized one (`from tests.testing_utils import get_sam_command`).

Reply:
Just to confirm (because I don't see it in the diff), We are still handling the `samdev` and `sam` correctly with init tests?

Reply:
Yes. There are 2 test files for `sam init`:
- [test_init_with_schemas_command.py](https://github.com/aws/aws-sam-cli/pull/3847/files#diff-54461e8d3ba983d84c58e6ab98cc5e1d2f9963e6dc0262a0f7a3057b06a46ddd) this file uses `CliRunner` from `click` library to pass input parameters for testing interactive behavior.
- [tests/integration/init/test_init_command.py](https://github.com/aws/aws-sam-cli/pull/3847/files#diff-02918e92551a578b2b429744d812bb4f971ec9cf18e6c4c4c9cd03a5ec9c2efe) this file mainly uses external process to run `samdev` or `sam`.  I removed method and use centralized one from here: https://github.com/aws/aws-sam-cli/blob/develop/tests/testing_utils.py#L41